README.md

```diff
@@ -3,40 +3,265 @@
 </a>
 
 # TwiML Generator
-A tool to help generate sample code for creating TwiML with Twilio's helper libraries
+A tool to help generate sample code for creating TwiML with Twilio's Helper Libraries
 
-## Supported Languages
+## Supported Helper Library Versions
 
-| Language  | SDK Version |
+| Helper Library  | Version |
 | :-------------  |:------------- |
-| C# | 5.x |
-| Java | 7.x |
-| Node | 3.x |
-| PHP | 5.x |
-| Python | 6.x |
+| [twilio-csharp](https://github.com/twilio/twilio-csharp/) | 5.x |
+| [twilio-java](https://github.com/twilio/twilio-java/) | 7.x |
+| [twilio-node](https://github.com/twilio/twilio-node/) | 3.x |
+| [twilio-php](https://github.com/twilio/twilio-php/) | 5.x |
+| [twilio-python](https://github.com/twilio/twilio-python/) | 6.x |
+| [twilio-ruby](https://github.com/twilio/twilio-ruby/) | ? |
```

Comment:
Thanks for adding -- I think this should be 5.x, it hasn't had a major version bump in years.

Reply:
BAHAHA just looked at the release versions and they're just rolling out a 6.x RC 🤦‍♀️ we're gonna need to do another version update with everything soon I guess.

Reply:
😭 I'll add that to the to-do list. Thanks!